packages/surveys/src/components/Headline.tsx

```diff
@@ -1,16 +1,31 @@
 interface HeadlineProps {
   headline?: string;
   questionId: string;
+  labelStyle?: any;
```

Comment:
Hey @gupta-piyush19 

Thanks for picking this up!

The only open question is why introduced this prop if it is not used. What is / was the idea for it?

Other than that, ready to merge :)

Reply:
@jobenjada , I added this to pass extra styles for thankyou card, but then used different approach. Removing it.